composables/IndyAPI.ts

```diff
@@ -99,11 +99,10 @@ export const GenerateVC = async (connectionID: string, credentialData: DiplomaSc
         'X-API-KEY': config.public.issuerAPIKey
       }
     })
-  console.log(data);
   return
 }
 
-export async function sendProofRequest (connectionID: string): Promise<void> {
+export async function sendProofRequest (connectionID: string): Promise<object> {
   const config = useRuntimeConfig()
   console.log('Sending proof request');
   const {data} = await axios.post(
```

Comment:
Can return the data directly - no need to use a variable.

Is there a definition for this data? Then it could be used in the return type of the method.

Reply:
No, there's no definition in the code, and since you mentioned it, I actually felt it was confusing. I'm only using the ID field from the response data object. 

Can you check this commit, I updated it in a way that only exposes the field that I'm using.  https://github.com/c4dt/eid-demo/pull/6/commits/66df23f52f211c72f5c59f60047a0928c7197382

Reply:
That simplifies it a lot. Great!